### PR TITLE
(docs): add emptyMessage

### PR DESCRIPTION
**Pull request by deeg**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: Docs

**What is the current behavior?** (You can also link to an open issue here)

`emptyMessage` was not documented in the docs.
 `loadingMessage` was defined in table options, but is never used.

**What is the new behavior?**

`emptyMessage` is documented in the docs
`loadingMessage` was removed from table options

I also sorted the table options docs page. Can revert that change if you'd like, but I find it easier to look things up.

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/api/options.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-* `columns`: Array of columns to display.
-* `scrollbarV`: Enable vertical scrollbar for fixed height vs fluid. This is necessary for virtual scrolling. Default value: `false`
-* `scrollbarH`: Enabled horiztonal scrollbars. Default value: `false`
-* `rowHeight`: The height of the row. This is necessary for virtual scrolling in order to calculate height for the scrollbar.
 * `columnMode`: The mode which the columns are distributed across the table. For example: `flex` will use flex-grow api, `force` will distribute proportionally and `standard` will just distrbute based on widths. Default value: `standard`
-* `headerHeight`: The height of the header in pixels. Pass a `falsey` for no header. Default value: `30`
-* `footerHeight`: The height of the footer in pixels. Pass a `falsey` for no footer. Default value: `0`
+* `columns`: Array of columns to display.
+* `count`: The total count of all rows. Default value: `0`
+* `emptyMessage`: The text to display when there is no data. Default value: 'No data to display'
```
Comment (deeg) [inline docs/api/options.md]: This is the only new one, besides the alpha sort changes.

File: src/models/table-options.model.ts [hunk 2]
```diff
@@ -53,9 +53,6 @@ export class TableOptions {
   // Page offset
   offset: number = 0;
 
-  // Loading indicator
-  loadingIndicator: boolean = false;
```
Comment (deeg) [inline src/models/table-options.model.ts]: This was not being used so I removed it to stop confusion from people. I can revert this change, if it is meant to be used somewhere.

Comment (deeg) [inline src/models/table-options.model.ts]: Sorry that was a mistake. Meant to delete loadingMessage, which is not used.

Will fix that and let you know when all set.

**Comment by deeg** [comment]
Sorry about spacing out there, the correct loadingMessage should be removed now.

**Comment by amcdnl** [comment]
Thanks!

**Pull request merged**